### PR TITLE
Add Damian to team

### PR DESCRIPTION
**Pull request by choldgraf**
This adds @damianavila to the 2i2c team webpage! 🎉🎉🎉

Here's the preview: https://deploy-preview-54--cocky-booth-e7ed17.netlify.app/about/

@damianavila I'll wait for you to 👍 before merging, to make sure the info looks OK

### CODE REVIEW

File: content/authors/damian-avila/_index.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+# Organizations/Affiliations
+organizations:
+- name: 2i2c
+  url: "https://2i2c.org"
+- name: Project Jupyter
+  url: "https://jupyter.org"
+
+# Short bio (displayed in user profile at end of posts)
+bio:
```
Comment (choldgraf) [inline content/authors/damian-avila/_index.md:23]: @damianavila - if you'd like, you can add a bio here!

Comment (damianavila) [reply content/authors/damian-avila/_index.md:23]: Great! I noticed others have the bio at the end of the `_index.md` file, should I drop it there (or here)?

Comment (choldgraf) [reply content/authors/damian-avila/_index.md:23]: ah you can put it at the end I think. If that's not right, we'll notice in the preview and can move it up here

File: content/authors/damian-avila/_index.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+# List each interest with a dash
+interests:
+
```
Comment (damianavila) [inline content/authors/damian-avila/_index.md:33]:
```suggestion
```suggestion
- Jupyter Ecosystem
- Finance/Quant
- Education
- Data Science
- Visualization
- Math/Statistics

```
```

File: content/authors/damian-avila/_index.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  link: https://github.com/damianavila
+
+# Enter email to display Gravatar (if Gravatar enabled in Config)
+# email: ""
+
+# Organizational groups that you belong to (for People widget)
+#   Set this to `[]` or comment out if you are not using People widget.
+user_groups:
+- Open Engineering Team
+---
```
Comment (damianavila) [inline content/authors/damian-avila/_index.md:56]:
```suggestion
```
```suggestion
---

Father, Software Developer, Quant, (formerly) Biochemist, and some other things ;-)
Currently living between Córdoba and Buenos Aires, Argentina.
I have made some contributions to popular Open Source projects such as Jupyter, Nikola, and Bokeh.
I have also started several projects being RISE (a "live" slideshow for the Jupyter notebook) the most popular one.
You can easily find videos of some of my talks and tutorials at multiples national and international conferences.
How can I help?
```

**Comment by damianavila** [comment]
@choldgraf tying the suggestion stuff I have not used before and I see you actively use it in other PRs :wink:
Let's see how that goes...

File: content/authors/damian-avila/_index.md (hunk 3 above)
Comment (damianavila) [reply content/authors/damian-avila/_index.md:56]: I am actually not sure why the platform added two suggestions here...

File: content/authors/damian-avila/_index.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+  url: "https://2i2c.org"
+- name: Project Jupyter
+  url: "https://jupyter.org"
+
+# Short bio (displayed in user profile at end of posts)
+bio:
+
+# List each interest with a dash
+interests:
+```suggestion
```
Comment (choldgraf) [inline content/authors/damian-avila/_index.md]:
```suggestion
```

File: content/authors/damian-avila/_index.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+  icon_pack: fab
+  link: https://github.com/damianavila
+
+# Enter email to display Gravatar (if Gravatar enabled in Config)
+# email: ""
+
+# Organizational groups that you belong to (for People widget)
+#   Set this to `[]` or comment out if you are not using People widget.
+user_groups:
+- Open Engineering Team
```
Comment (choldgraf) [inline content/authors/damian-avila/_index.md:55]:
```suggestion
- Open Engineering Team
---

Father, Software Developer, Quant, (formerly) Biochemist, and some other things ;-)
Currently living between Córdoba and Buenos Aires, Argentina.
I have made some contributions to popular Open Source projects such as Jupyter, Nikola, and Bokeh.
I have also started several projects being RISE (a "live" slideshow for the Jupyter notebook) the most popular one.
You can easily find videos of some of my talks and tutorials at multiples national and international conferences.
How can I help?
```

**Comment by choldgraf** [comment]
Looks great! 🚀 

wohoo it's public!

**Pull request merged**